docs/en/sql-reference/dictionaries/external-dictionaries/external-dicts.md

```diff
@@ -24,6 +27,16 @@ The [dictionaries](../../../operations/system-tables/dictionaries.md#system_tabl
 -   Configuration parameters.
 -   Metrics like amount of RAM allocated for the dictionary or a number of queries since the dictionary was successfully loaded.
 
+## Creating a dictionary with a DDL query
+
+Dictionaries can be created with [DDL queries](../../../sql-reference/statements/create/dictionary.md).  This does not require any additional records in a server configuration file. This allows dictionaries to be worked with as first-class entities, like tables or views.
```

Comment:
I like that, added like so:
## Creating a dictionary with a DDL query                                                                                 
                                                                                                                              
Dictionaries can be created with [DDL queries](../../../sql-reference/statements/create/dictionary.md), and this is the recommended method because with DDL created dictionaries:                                                                   
 - No additional records are added to server configuration files                                                           
 - The dictionaries can be worked with as first-class entities, like tables or views                                       
 - Data can be read directly, using familiar SELECT rather than dictionary table functions                                 
 - The dictionaries can be easily renamed